### PR TITLE
UIOR-673 Print order function

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
https://issues.folio.org/browse/UIOR-673
https://issues.folio.org/browse/UIOR-672

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.colHead {
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.css:1]: looks like it's not used with blocks approach

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        ? `${value}%`
+        : (
+          <AmountWithCurrencyField
+            currency={currency}
+            amount={value}
+          />
+        );
+    default:
+      if (value === true) {
+        return 'true';
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.js:53]: should it be translated?

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            <div>Phone: {dataSource.vendorPrimaryPhone?.phoneNumber}</div>
+          </Col>
+          <Col xs={6}>
+            <div>Ship to:</div>
+            <div>{dataSource.shipToAddress}</div>
+          </Col>
+        </Row>
+
+      </Grid>
+      <Row className={css.space}>------------------------------------</Row>
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.js:103]: is it smth to be removed?

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+      return value ?? <NoValue />;
+  }
+}
+
+const ComponentToPrint = ({ dataSource = {} }) => {
+  return (
+    <div>
+      <Grid>
+        <Row>
+          <Col xs={6}>Lib name</Col>
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.js:67]: are translations required?

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/PrintOrder/ComponentToPrint/ComponentToPrint.js:53]: It's a good question what level of translation is needed. For example, Chinese labels might not help if they are sent to english speaking vendor

**Review by axelhunn — APPROVED** [review]

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.colB {
+  background: rgba(0, 0, 0, 0.08);
+}
+
+.dataRow {
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.css:5]: looks like not used

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.colB {
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.css:1]: can we use more suitable name? I used just to not spend time
mb `orderLineDetails`?

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+            <div>
+              <FormattedMessage id="ui-orders.orderSummary.workflowStatus" />:&nbsp;
+              {dataSource.workflowStatus ? ORDER_STATUS_LABEL[dataSource.workflowStatus] : <NoValue />}
+            </div>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={6}>
+            <div>
+              <FormattedMessage id="ui-orders.orderSummary.closingReason" />:&nbsp;
```
Comment (NikitaSedyx) [inline src/PrintOrder/ComponentToPrint/ComponentToPrint.js:90]: don't we have keyvalue but for horizontal?

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/PrintOrder/ComponentToPrint/ComponentToPrint.js]: added translation

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js (hunk 4 above)
Comment (aliaksei-chumakou) [reply src/PrintOrder/ComponentToPrint/ComponentToPrint.js]: added tranlsations

**Review by NikitaSedyx — APPROVED** [review]
the rest is lgtm, the only question about horizontal key value, here we copy paste Label + : + nbsp + structure + NoValue, and the same will be for vouchers

File: src/PrintOrder/ComponentToPrint/ComponentToPrint.js (hunk 7 above)
Comment (aliaksei-chumakou) [reply src/PrintOrder/ComponentToPrint/ComponentToPrint.js:90]: I think no, we don't have such one. Also I'm not sure that we need to make labels in bold.

**Pull request merged**